error/ApiError.js

```diff
@@ -0,0 +1,17 @@
+class ApiError {
+
+    constructor(code, message) {
+        this.code  = code;
+        this.message = message;
+    }
+
+    static badRequest(message) {
```

Comment:
I don't understand why we need class to create more instances of class via static methods. It is better to have a file with functions of multiple classes (one for each error type) and use them directly. Can you please update it?